mmdet/core/evaluation/text_evaluation.py

```diff
@@ -16,19 +16,28 @@
 import cv2
 import matplotlib.pyplot as plt
 import numpy as np
-import Polygon as plg
+
+# ATTENTION! the obsolete Polygon3 was replaced with shapely.geometry.Polygon,
+#            but the changes were not debugged properly.
```

Comment:
"Debugged" kinda hints that there are bugs and we know about them but won't fix. Was the logic involved tested somehow?

Reply:
Since Ilya checked the code, I removed this comment